advanced/spec/binarySearchTree_test.js

```diff
@@ -0,0 +1,31 @@
+import chai, { expect } from 'chai'
+import chaiChange from 'chai-change'
+import BinarySearchTree from '../src/binarySearchTree'
+
+chai.use(chaiChange)
+
+describe('BinarySearchTree', () => {
+'use strict'
+
```

Comment:
You may want to use a beforeEach function to create the instance of BinarySearchTree and even to populate it with data. The beforeEach function runs before each test. Format would be something like:
```
let bst

beforeEach( function () {
   bst = new BinarySearchTree() 
   bst.insert( someData )    // If you would like to populate each instance with some data
})
```